cve_bin_tool/sbom_manager/__init__.py

```diff
@@ -44,6 +45,33 @@ def __init__(
         # Connect to the database
         self.cvedb = CVEDB(version_check=False)
 
+    def find_product_location(self, product_name):
```

Comment:
So other than the logical implementation of the function that i don't know much about, i have a nitpick to make here, if your function isn't actually utilizing anything from the class itself why not make it a standalone function instead of member function and it probably should go inside the utils.py, similarly the function is same for the version_scanner.py there too it being a member function it can be reuse from the utils.py instead.

Reply:
@mastersans Understandable, it would be much more cleaner. I worked very roughly to figure out things, missed to organize it well in the end. Thank you for the suggestion. I will update it 👍